bucket/openttd.json

```diff
@@ -1,48 +1,65 @@
     "description": "Simulation game based upon Transport Tycoon Deluxe",
-    "version": "1.10.0",
+    "version": "12.0",
     "license": "GPL-2.0-or-later",
     "architecture": {
-        "64bit": {
-            "url": [
-                "https://cdn.openttd.org/openttd-releases/1.10.0/openttd-1.10.0-windows-win64.zip",
-                "https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip",
-                "https://cdn.openttd.org/opensfx-releases/0.2.3/opensfx-0.2.3-all.zip",
-                "https://cdn.openttd.org/openmsx-releases/0.3.1/openmsx-0.3.1-all.zip"
-            ],
-            "hash": [
-                "263f0b51d9630a1ef60538e822de064138ea3599f1ace4cc070943e81aa5bc4d",
-                "d419c0f5f22131de15f66ebefde464df3b34eb10e0645fe218c59cbc26c20774",
-                "6831b651b3dc8b494026f7277989a1d757961b67c17b75d3c2e097451f75af02",
-                "92e293ae89f13ad679f43185e83fb81fb8cad47fe63f4af3d3d9f955130460f5"
-            ]
-        },
         "32bit": {
-            "url": [
-                "https://cdn.openttd.org/openttd-releases/1.10.0/openttd-1.10.0-windows-win32.zip",
-                "https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip",
-                "https://cdn.openttd.org/opensfx-releases/0.2.3/opensfx-0.2.3-all.zip",
-                "https://cdn.openttd.org/openmsx-releases/0.3.1/openmsx-0.3.1-all.zip"
-            ],
-            "hash": [
-                "4f10643582e0c57afeb8b327364d4ff20f3c28e10399d8be7ddf83104b3806cf",
-                "d419c0f5f22131de15f66ebefde464df3b34eb10e0645fe218c59cbc26c20774",
-                "6831b651b3dc8b494026f7277989a1d757961b67c17b75d3c2e097451f75af02",
-                "92e293ae89f13ad679f43185e83fb81fb8cad47fe63f4af3d3d9f955130460f5"
-            ]
+            "url": "https://cdn.openttd.org/openttd-releases/12.0/openttd-12.0-windows-win32.zip",
+            "hash": "1086f94a9888a9e77c5201d0a2612d1fcd6d45b49acb4250394b073917f309ca",
+            "extract_dir": "openttd-12.0-windows-win32"
+        },
+        "64bit": {
+            "url": "https://cdn.openttd.org/openttd-releases/12.0/openttd-12.0-windows-win64.zip",
+            "hash": "872afdbba74fce18dd6f0d3ba2a7baef8800fe0ef486b383b997a0f3b285c7a4",
+            "extract_dir": "openttd-12.0-windows-win64"
         }
     },
-    "pre_install": [
-        "Move-Item \"$dir\\opengfx-0.6.0.tar\" \"$dir\\baseset\"",
-        "Move-Item \"$dir\\opensfx-0.2.3\" \"$dir\\baseset\"",
-        "Move-Item \"$dir\\openmsx-0.3.1\" \"$dir\\baseset\""
+    "post_install": [
+        "if (!(Test-Path \"$persist_dir\\openttd.cfg\")) {",
+        "   New-item -ItemType File \"$persist_dir\\openttd.cfg\" | Out-Null",
+        "}"
+    ],
+    "persist": [
+        "content_download",
+        "newgrf",
+        "save",
+        "scenario",
+        "screenshot"
+    ],
+    "bin": [
+        [
+            "openttd.exe",
+            "openttd-persist",
```

Comment:
Why not just call it `openttd` instead of `openttd-persist`? The shortcut entry below isn't added to the `PATH` automatically.

Reply:
It was the manifest I started with, so I wasn't sure if I could add an option and use the same name.
Use the same name as other manifests.